src/_includes/head.html

```diff
@@ -0,0 +1,28 @@
+<html lang="en-gb">
```

Comment:
It'll still get closed correctly, even though it gets opened in this partial. Could add a comment above the closing tag for readability so it's known this is where it's closing and that the opening tag is elsewhere.

TBH same with the `DOCTYPE` so it could've stayed in this file